### DISCUSSION TITLE
CUDA Programming

### DISCUSSION THREAD

**Discussion by JoelMathewC** [bfGraph/STGraph#52] [General]
Good resource to get started: https://developer.nvidia.com/blog/tag/cuda-refresher/

**Comment by nithinmanoj10** [comment]
# Basics of CUDA Programming

## L
[Launch Kernel Parameters](https://medium.com/analytics-vidhya/cuda-compute-unified-device-architecture-part-2-f3841c25375e)

## T
[Thread Indexing Cheatsheet](https://cs.calvin.edu/courses/cs/374/CUDA/CUDA-Thread-Indexing-Cheatsheet.pdf)

**Comment by JoelMathewC** [comment]
## Note
1. Threads in a grid execute the same kernel
2. A grid contains blocks and a block contains threads

Important reference: https://www.tutorialspoint.com/cuda/cuda_keywords_and_thread_organization.htm

**Comment by JoelMathewC** [comment]
Just a learning discussion. Closing this out since there isn't a need for this to be left open.
